vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridCellFocusPageIT.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.grid.it;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.vaadin.flow.component.grid.CellFocusEvent;
+import com.vaadin.flow.component.grid.testbench.GridElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.tests.AbstractComponentIT;
+
+/**
+ * IT for grid's Flow based cell focus event.
+ *
+ * @author Vaadin Ltd
+ */
+
+
+@TestPath("vaadin-grid/grid-cell-focus-page")
+public class GridCellFocusPageIT extends AbstractComponentIT {
+
+    @Test
+    public void focusBodyCell() {
+        open();
+
+        getGrid().getCell(0, 0).focus();
```

Comment:
The Selenium focus call was not reliable and was changed a few times until it behaved consistently. The last related change is this one: https://github.com/vaadin/flow-components/pull/1802

In the future, please make sure you are porting the latest version of the code, rather than just cherry-picking the commit.

Reply:
I changed it back to click(0,0). Thank you.